src/Hazelcast.Net/Clustering/LoadBalancing/MembersInfoCollection.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using Hazelcast.Models;
+
+namespace Hazelcast.Clustering.LoadBalancing
+{
+    public class MembersInfoCollection
```

Comment:
Since this is a collection of `MemberInfo` can we name it `MemberInfoCollection`? Without an "s" after "Member"?